voidcrew/modules/ship_parts/user_prefs.dm

```diff
@@ -0,0 +1,27 @@
+/datum/preferences
+
+	///Ships owned by the owner of the prefs
+	var/list/ships_owned = list(
+		/obj/item/ship_parts/neutral = 0,
+		/obj/item/ship_parts/nanotrasen = 0,
+		/obj/item/ship_parts/syndicate = 0,
+	)
+
+/datum/preferences/proc/save_ships()
+	if(!path)
+		return FALSE
+	if(!fexists(path))
+		return FALSE
+
+	var/savefile/S = new /savefile(path)
```

Comment:
might want to put this on hold for https://github.com/tgstation/tgstation/pull/70492 so you can leverage that new API

Reply:
damn it will be swag when we get it